src/CmsInterfaceManagerBundle/objects/db/TCMSInterfaceManager/TCMSInterfaceManagerBaseExportCSV.class.php

```diff
@@ -108,14 +108,15 @@ protected function PrepareImport()
      *
      * @param bool $bImportSucceeded - set to true if the import succeeded
      *
-     * @FIXME Parent return `$bImportSucceeded` (bool). This method should probably do the same?
-     * @return void
+     * @return bool
      */
     protected function Cleanup($bImportSucceeded)
     {
         parent::Cleanup($bImportSucceeded);
         $query = 'DROP TABLE `'.$this->GetExportTableName().'`';
         MySqlLegacySupport::getInstance()->query($query);
+
+        return true;
```

Comment:
This method had no return statement before but it should be returning a bool (according to it's parent class).
The return type is never checked, so this does not change behaviour